[2022-10-04 13:29:45,701] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_star_wars.get_raw_star_wars_people_data scheduled__2022-10-03T00:00:00+00:00 [queued]>
[2022-10-04 13:29:45,714] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_star_wars.get_raw_star_wars_people_data scheduled__2022-10-03T00:00:00+00:00 [queued]>
[2022-10-04 13:29:45,715] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-10-04 13:29:45,716] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-10-04 13:29:45,717] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-10-04 13:29:45,730] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_raw_star_wars_people_data> on 2022-10-03 00:00:00+00:00
[2022-10-04 13:29:45,738] {standard_task_runner.py:52} INFO - Started process 3882 to run task
[2022-10-04 13:29:45,741] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ingest_star_wars', 'get_raw_star_wars_people_data', 'scheduled__2022-10-03T00:00:00+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/ingest_starwars_data.py', '--cfg-path', '/tmp/tmp20sz2rvv', '--error-file', '/tmp/tmppqqrbe_5']
[2022-10-04 13:29:45,745] {standard_task_runner.py:77} INFO - Job 42: Subtask get_raw_star_wars_people_data
[2022-10-04 13:29:45,802] {logging_mixin.py:109} INFO - Running <TaskInstance: ingest_star_wars.get_raw_star_wars_people_data scheduled__2022-10-03T00:00:00+00:00 [running]> on host be1a5b89e146
[2022-10-04 13:29:45,856] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=xxxx@xxxxx.com
AIRFLOW_CTX_DAG_OWNER=Ram
AIRFLOW_CTX_DAG_ID=ingest_star_wars
AIRFLOW_CTX_TASK_ID=get_raw_star_wars_people_data
AIRFLOW_CTX_EXECUTION_DATE=2022-10-03T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-03T00:00:00+00:00
[2022-10-04 13:29:54,515] {python.py:175} INFO - Done. Returned value was: None
[2022-10-04 13:29:54,576] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ingest_star_wars, task_id=get_raw_star_wars_people_data, execution_date=20221003T000000, start_date=20221004T132945, end_date=20221004T132954
[2022-10-04 13:29:54,647] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-10-04 13:29:54,689] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
